nucypher/blockchain/eth/actors.py

```diff
@@ -1086,6 +1095,29 @@ def disable_winding_down(self) -> dict:
         receipt = self._set_winding_down(value=False)
         return receipt
 
+    def non_withdrawable_stake(self) -> NU:
+        staked_amount: NuNits = self.staking_agent.non_withdrawable_stake(staker_address=self.checksum_address)
+        return NU.from_nunits(staked_amount)
+
+    def mintable_periods(self) -> int:
+        """
+        Returns number of periods that can be rewarded in the current period. Value in range [0, 2]
+        """
+        current_period: Period = self.staking_agent.get_current_period()
+        previous_period: int = current_period - 1
+        current_committed_period: Period = self.staking_agent.get_current_committed_period(staker_address=self.checksum_address)
+        next_committed_period: Period = self.staking_agent.get_next_committed_period(staker_address=self.checksum_address)
+
+        mintable_periods: int = 0
+        for committed_period in (current_committed_period, next_committed_period):
+            if committed_period == 0:
+                continue
+            if committed_period <= previous_period:
+                mintable_periods += 1
```

Comment:
```suggestion
        if 0 < current_committed_period <= previous_period:
            mintable_periods += 1
        if 0 < next_committed_period <= previous_period:
            mintable_periods += 1
```
I think that's clearer, but I don't know if that captures your original intent.

Reply:
✔️